source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/admin/settings/organizations/edit/index.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import * as Toast from '@lib/toast';
+import { compose } from 'recompose';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { withData as withOrbit } from 'react-orbitjs';
+
+import {
+  withDataActions,
+  IProvidedProps as IOrganizationProps
+} from '@data/containers/resources/organization/with-data-actions';
+
+import OrganizationForm from '../common/form';
+
+import { listPathName } from '../index';
+import { query, withLoader, buildOptions, buildFindRecord, OrganizationResource, UserResource } from '@data';
+
+interface IOwnProps {
+  organization: OrganizationResource;
+  owner: UserResource;
+}
+
+type IProps =
+  & IOrganizationProps
+  & IOwnProps
+  & RouteComponentProps<{}>;
+
+class EditOrganization extends React.Component<IProps> {
+
+  update = async (attributes, onSuccess) => {
+
+    const { updateAttributes, updateOwner } = this.props;
+
+    try {
+      await updateAttributes({
+        name: attributes.name,
+        websiteUrl: attributes.websiteUrl,
```

Comment:
this can all be replaced with `await updateAttributes(attributes)`?

Reply:
No, because im sending the state which other stuff that isnt attributes

Reply:
so then it seems like a separation of concerns violation to this